### PR TITLE
AzureTRE Release 0.5 alignment

### LINKED ISSUE #26: MGMT_RESOURCE_GROUP var name is not consistent
Management resource group var name is inconsistent. In some places it is MGMT_RESOURCE_GROUP and some MGMT_RESOURCE_GROUP_NAME. Should be MGMT_RESOURCE_GROUP_NAME in all places.

### PR DESCRIPTION
**Pull request by LizaShak**
## What is being addressed

Fixes #26.
Align this repo to changes in AzureTRE release 0. 5

## How is this addressed

- rename all the instances of the env var MGMT_RESOURCE_GROUP to be MGMT_RESOURCE_GROUP_NAME

### CODE REVIEW

File: .github/workflows/deploy_tre.yml [hunk 1]
```diff
@@ -46,7 +46,7 @@ jobs:
       ARM_TENANT_ID: ${{ secrets.ARM_TENANT_ID }}
       CORE_ADDRESS_SPACE: ${{ secrets.CORE_ADDRESS_SPACE }}
       LOCATION: ${{ secrets.LOCATION }}
-      MGMT_RESOURCE_GROUP: ${{ secrets.MGMT_RESOURCE_GROUP }}
+      MGMT_RESOURCE_GROUP_NAME: ${{ secrets.MGMT_RESOURCE_GROUP_NAME }}
       MS_TEAMS_WEBHOOK_URI: ${{ secrets.MS_TEAMS_WEBHOOK_URI }}
       STATE_STORAGE_ACCOUNT_NAME: ${{ secrets.STATE_STORAGE_ACCOUNT_NAME }}
```
[the remark this answers is not in the document]
Comment (tamirkamara) [reply .github/workflows/deploy_tre.yml]: done

File: .devcontainer/devcontainer.json [hunk 2]
```diff
@@ -11,7 +11,8 @@
       // 		export DOCKER_GROUP_ID=$(getent group docker | awk -F ":" '{ print $3 }')
       "DOCKER_GROUP_ID": "${localEnv:DOCKER_GROUP_ID}",
       "INTERACTIVE": "true",
-      "OSS_VERSION": "v0.4.3"
+      // TODO: replace with v0.5.0 when available
+      "OSS_VERSION": "349c42469f6896826db38f97afaae208b984a96a"
```
Comment (tamirkamara) [inline .devcontainer/devcontainer.json:15]: This has to be done before merging

Comment (marrobi) [reply .devcontainer/devcontainer.json]: Done.

**Comment by marrobi** [comment]
Closing as superceeded by 0.5.1

**Pull request closed without merging**